src/Microsoft.TestPlatform.Common/Resources/Resources.Designer.cs

```diff
@@ -159,20 +159,18 @@ internal class Resources {
                 return ResourceManager.GetString("FailedToFindInstalledUnitTestExtensions", resourceCulture);
             }
         }
-
+        
         /// <summary>
         ///   Looks up a localized string similar to An error occured while creating Fast filter..
         /// </summary>
-        internal static string FastFilterException
-        {
-            get
-            {
+        internal static string FastFilterException {
+            get {
                 return ResourceManager.GetString("FastFilterException", resourceCulture);
             }
         }
-
+        
         /// <summary>
-        ///   Looks up a localized string similar to There are multiple configurations that have diagnostic data adapter  type &apos;{0}&apos; or Uri &apos;{1}&apos;. Duplicate configurations will be ignored in the test run..
+        ///   Looks up a localized string similar to There are multiple configurations that have data collector FriendlyName as &apos;{0}&apos;. Duplicate configurations will be ignored in the test run..
```

Comment:
this removed a `{1}` too - Might want to edit the Format message?

Reply:
Hmm, those aren't from my change, not sure what happened here...